archunit/src/test/java/com/tngtech/archunit/core/domain/DependencyTest.java

```diff
@@ -248,6 +262,14 @@ void method() throws IOException {
         }
     }
 
+    private static class ClassWithDependencyOnInstanceofCheck {
```

Comment:
Would be nice to add some @SuppressWarnings("unused") here like for the other little test class snippets, so the IDE doesn't always give false positive warnings.